views/marts/mlt/fct_inapp_trial_requests.view.lkml

```diff
@@ -0,0 +1,218 @@
+  }
+
+  dimension: country_name {
+    type: string
+    sql: ${TABLE}.country_name ;;
+    label: "Country"
+    description: "The name of the country, as filled in in the trial request from."
+  }
+
+  dimension: name {
+    type: string
+    sql: ${TABLE}.name ;;
+    label: "Name"
+    description: "The display name of the user who requested the trial."
+  }
+
+  dimension: request_source {
+    type: string
+    sql: ${TABLE}.request_source ;;
+    label: "Request source"
+    description: "Whether this trial request is originating from In-app or from Mattermost website."
+  }
+
+  dimension: site_name {
+    type: string
+    sql: ${TABLE}.site_name ;;
+    label: "Site Name"
+    description: "The name of the Mattermost installation, as defined in the administrator view."
+  }
+
+  dimension: site_url {
+    type: string
+    sql: ${TABLE}.site_url ;;
+    label: "Site URL"
+    description: "The URL of the installation set via the system console in Mattermost by the user, or https://mattermost.com if trial was requested from mattermost.com/trial."
+  }
+
+  dimension: num_users {
+    type: number
+    sql: ${TABLE}.num_users ;;
+    label: "Number of users"
+    description: "The number of users."
+  }
+
+
+  dimension: is_valid_trial_email {
+    type: yesno
+    sql: ${TABLE}.is_valid_trial_email ;;
+    label: "Is Valid Email?"
+    description: "Whether the current email is valid. Emails are considered valid if they match format `%_@__%.__%`` and don't contain any of the characters `'\/()&$^!`."
+  }
+
+  dimension: is_first_trial {
+    type: yesno
+    sql: ${TABLE}.is_first_trial ;;
+    label: "Is First Trial?"
+    description: "Whether this is the first trial (in chronological order) requested by this email."
+  }
+
+  dimension: is_last_trial {
+    type: yesno
+    sql: ${TABLE}.is_last_trial ;;
+    label: "Is last trial?"
+    description: "Whether this is the first trial (in chronological order) requested by this email."
+  }
+
+
+  ###
+  ### Dates for the current trial
+  ###
+
+  dimension_group: start_at {
+    type: time
+    timeframes: [raw, date, week, month, quarter, year]
+    sql: CAST(${TABLE}.start_at AS TIMESTAMP_NTZ) ;;
+    label: "Start At"
+    description: "Date when trial starts at. Note that trials start as soon as the trial request is created, making this date similar to trial creation timestamp."
+  }
+
+
+  dimension_group: end {
+    type: time
+    timeframes: [raw, date, week, month, quarter, year]
+    sql: CAST(${TABLE}.end_at AS TIMESTAMP_NTZ) ;;
+    label: "End At"
+    description: "Date when trial ends at."
+  }
+
+
+  ###
+  ### Extra dimensions
+  ###
+
+
+  dimension: contact_email {
+    type: string
+    sql: ${TABLE}.contact_email ;;
+    label: "Contact Email"
+    description: "The contact email, as filled in in the input box in the trial request form Prefer trial email."
```

Comment:
"Prefer trial email" ?

Reply:
There's a missing `.`. I added these for debugging purposes, and that's the reason I added the suggestion in the description.

To add more context,  @nickmisasi  suggested that we should coalesce `contact_email` into `email`.  They will either be the same or `contact_email` will be missing and email will be populated. However this is not true, as there are cases where contact email is != than email. So I thought of adding both the coalesced value and the original values (with an asterisk) so that we can debug.